radicle-node/src/service.rs

```diff
@@ -394,36 +394,44 @@ where
 
         match cmd {
             Command::Connect(id, addr) => self.reactor.connect(id, addr),
-            Command::Fetch(id, resp) => {
+            Command::Fetch(rid, resp) => {
                 if !self
                     .tracking
-                    .is_repo_tracked(&id)
+                    .is_repo_tracked(&rid)
```

Comment:
If adding context is the goal, I'm unsure if the rename is much more helpful.  The documentation says its a project ID, maybe 'project_id' would be better? 'rid' sounds like 'repo id' or 'remote id'.

```
    /// Fetch the given project from the network.
    Fetch(Id, chan::Sender<FetchLookup>),
```

Reply:
RID stands for repo id indeed, I'm slowly renaming stuff as I go.

Reply:
Cool, looks like the documentation would be updated with this change